packages/components/src/higher-order/with-focus-outside/test/index.tsx

```diff
@@ -92,32 +92,4 @@ describe( 'withFocusOutside', () => {
 
 		await waitFor( () => expect( onFocusOutside ).toHaveBeenCalled() );
 	} );
-
-	it( 'should not call handler if focus shifts outside the component when the document does not have focus', async () => {
```

Comment:
This test is an implementation detail on useFocusOutside which is no longer valid. We don't test for document.hasFocus()